react-refresher/my-app/src/Counter.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+export default function Counter({ initialNum }) {
+  const [count, setCount] = useState(Number(initialNum));
+  const handleCount = (event) => {
```

Comment:
My preference is to use `function syntax` for named functions instead of `arrow-function syntax`. I think it's easier to read and understand. But, that's just my personal preference. There are many JS developers who use arrow function syntax for everything.